server/models/Event.js

```diff
@@ -1,4 +1,7 @@
-const { Schema, model } = require("mongoose");
+// const { Schema, model } = require("mongoose");
+
+const mongoose = require('mongoose');
+const { Schema, model } = mongoose;
```

Comment:
I think this can all get consolidated to this:

```suggestion
const { Schema, model } = require("mongoose");
```